javascript-loops-practice/add-suffix-to-all.js

```diff
@@ -0,0 +1,8 @@
+/* exported addSuffixToAll */
+function addSuffixToAll(words, suffix) {
+  var suffixArray = [];
+  for (var i = 0; i < words.length; i++) {
+    suffixArray.push(words[i] += suffix);
```

Comment:
This is making a change to the original array, which is not good. Instead, you should just use that original array to generate the new values, but leave the original intact.